pkg/dependencies/min_version.go

```diff
@@ -0,0 +1,42 @@
+// Copyright (C) 2022, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+package dependencies
+
+import (
+	"encoding/json"
+	"fmt"
+	"strconv"
+
+	"golang.org/x/mod/semver"
+
+	"github.com/ava-labs/avalanche-cli/pkg/models"
+
+	"github.com/ava-labs/avalanche-cli/pkg/application"
+	"github.com/ava-labs/avalanche-cli/pkg/constants"
+)
+
+func CheckVersionIsOverMin(app *application.Avalanche, dependencyName string, network models.Network, version string) error {
+	dependencyBytes, err := app.Downloader.Download(constants.CLILatestDependencyURL)
```

Comment:
should you just return nil if version is empty? eg local networks

Reply:
Its not returning error right now if its local networks. The test in min_version_test for local network shows that.